Updates to documentation of two tf.contrib.metrics functions.

kbrose opened the pull request:
- Added code wrapping markdown for function `tf.contrib.metrics.aggregate_metrics`.
- General editing of `tf.contrib.metrics.confusion_matrix` for clarity.

kbrose committed:
updating tf.contrib.metrics.confusion_matrix documentation

kbrose committed:
tf.contrib.metrics.aggregate_metrics doc missing code markdown

tensorflow-jenkins commented:
Can one of the admins verify this patch?

rohan100jain commented on tensorflow/contrib/metrics/python/ops/confusion_matrix_ops.py:
````diff
@@ -33,26 +33,27 @@ def confusion_matrix(predictions, labels, num_classes=None, dtype=dtypes.int32,
+       [0, 0, 1, 0]
+       [0, 0, 1, 0]
+       [0, 0, 0, 1]]
+  ```
````
Can you add a summary line here saying that since the highest value label was 3, the label space assumed is 0, 1, 2, 3 and hence we end up with n=4 and a 4X4 matrix. I feel that'll make this clearer.

rohan100jain reviewed (changes requested):
Overall its good. Just a minor comment.

kbrose committed:
clarifying how num_classes works w.r.t. assumed possible classes

kbrose commented:
@rohan100jain Added new commit, updated description to clarify how `num_classes` is computed, and updated example code to hopefully clarify further what labels are assumed to exist.

rohan100jain reviewed (approved):
Looks great!

rohan100jain commented:
@tensorflow-jenkins test this please
